docs/resources/integration_webhook.md

```diff
@@ -8,18 +8,18 @@ description: |-
 
 # wallarm_integration_webhook
 
-Provides the resource to manage integrations via generic webhooks. Webhooks can be used as system log sources. The number of log sources depends on the system complexity: the more components in the system, the greater number of log sources and logs.
+Provides the resource to manage [integrations via generic webhooks][1]. Webhooks can be used as system log sources. The number of log sources depends on the system complexity: the more components in the system, the greater number of log sources and logs.
```

Comment:
hmmm, strange description for this case, why do we need to mention "The number of log sources depends on the system complexity: the more components in the system, the greater number of log sources and logs" here?